packages/beeq/src/components/card/_storybook/bq-card.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import mdx from './bq-card.mdx';
+import { CARD_BORDER_RADIUS, CARD_TYPE } from '../bq-card.types';
+
+const meta: Meta = {
+  title: 'Components/Card',
+  component: 'bq-card',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    border: { control: 'select', options: [...CARD_BORDER_RADIUS] },
+    content: { control: 'text', table: { disable: true } },
+    type: { control: 'select', options: [...CARD_TYPE] },
```

Comment:
Please move the border and the type control props on top and add a comment to the content prop so that we know is not part of the component 😉 

```suggestion
    border: { control: 'select', options: [...CARD_BORDER_RADIUS] },
    type: { control: 'select', options: [...CARD_TYPE] },
    // This control is not part of the component
    content: { control: 'text', table: { disable: true } },
```